components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -600,6 +603,105 @@ private static void buildSCIMProvisioningConfiguration(IdentityProvider fedIdp,
 
     }
 
+    /**
+     * @param fedIdp
+     * @param paramMap
+     * @throws IdentityApplicationManagementException
+     */
+    private static void buildSCIM2ProvisioningConfiguration(IdentityProvider fedIdp,
+                                                           Map<String, String> paramMap)
+            throws IdentityApplicationManagementException {
+
+        ProvisioningConnectorConfig proConnector = new ProvisioningConnectorConfig();
+        proConnector.setName("SCIM2");
+
+        Property userNameProp = null;
+        Property passwordProp = null;
+        Property userEpProp = null;
+        Property groupEpProp = null;
+        Property scim2UserStoreDomain = null;
+        Property scim2EnablePwdProvisioning = null;
+        Property defaultPwdProp = null;
+        Property uniqueID = null;
+
+        if (paramMap.get("scim2ProvEnabled") != null && "on".equals(paramMap.get("scim2ProvEnabled"))) {
```

Comment:
better to assign `paramMap.get("scim2ProvEnabled")` to a variable and resuse

Reply:
`paramMap.get("scim2ProvEnabled")` is only used twice so it would not be much useful to assign a separate variable.